src/clj/harja/kyselyt/urakat.sql

```diff
@@ -533,3 +532,24 @@ WHERE st_dwithin(alue, st_makepoint(:x, :y), :treshold);
 -- name: luo-paallystyspalvelusopimus<!
 INSERT INTO paallystyspalvelusopimus (alueurakkanro, alue, paallystyspalvelusopimusnro)
 VALUES (:alueurakkanro, ST_GeomFromText(:alue) :: GEOMETRY, :paallystyssopimus);
+
+-- name: hae-lahin-hoidon-alueurakka
+SELECT
+  id,
+  etaisyys
+FROM (SELECT
+        u.id,
+        st_distance(au.alue, st_makepoint(:x,:y)) AS etaisyys
+      FROM urakka u
+        JOIN alueurakka au ON au.alueurakkanro = u.urakkanro
+      WHERE
+        u.alkupvm <= now() AND
+        u.loppupvm > now()
```

Comment:
Etäisyys WHERE suoraan tähän niin ei tarvitse sisäkkäistä kyselyä.
AND st_distance(au.alue, st_makepoint(:x,:y)) < :maksimietaisyys